### DISCUSSION TITLE
Window blacklist / whitelist

### DISCUSSION THREAD

**Discussion by KCGD** [gTile/gTile#393] [Ideas]
I have one specific window that i do not want to tile. Is it possible to add it to some sort of black list that the tiler will ignore?

**Comment by scherepanov** [comment]
It would be a nice feature.
And, it actually exist in original @vibou code. At time when I got my hands on gTile, this feature did not function, and it was removed.
It is not a highly requested feature, you are the first one to ask in many years.
If you feel like doing it, feel free to do some coding and submit PR, your contributions are welcome.
